client/src/modules/stock/StockForm.service.js

```diff
@@ -125,5 +126,57 @@ function StockFormService(StockItem, Store, AppCache, Session, $timeout) {
     return Object.keys(this.cache).length > 0;
   };
 
+  /**
+   * @method catchDuplicatedLots
+   *
+   * @description
+   * this method catch duplicated row and emit notification on the row
+   */
+  StockForm.prototype.catchDuplicatedLots = function catchDuplicatedLots(_store) {
```

Comment:
Since this returns a boolean, can we call it `hasDuplicatedLots`?

Reply:
Also, I think you can just use `this.store` instead of passing it in, no?

Reply:
I have tried with `this.store` it doesn't work curiously

Reply:
It works 😆